modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -40,72 +40,172 @@ or the xref:{version-server}@server:manage:manage-settings/install-sample-bucket
+For the purposes of this example, we are connecting to Couchbase using the _Full Admin_ role created during the installation of our Couchbase Server.
+
+Couchbase RBAC is fully described in the xref:{version-server}@server:manage:manage-security/manage-users-and-roles.adoc[Manage Users, Groups, and Roles] page.
+--
+====
+
+Following successful authentication, the bucket can be opened:
+
+[source,golang]
+----
+include::devguide:example$go/cloud.go[tag=bucket,indent=0]
+----
+
+NOTE: We are working with the xref:{version-server}@server:manage:manage-settings/install-sample-buckets.adoc[travel-sample] data bucket. 
+If you are not, update the `bucketName` variable used in the example with your own.
+
+The {go-current-version} SDK supports full integration with the xref:concept-docs:collections.adoc[Collections] feature introduced in Couchbase Server 7.0.
+_Collections_ allow documents to be grouped by purpose or theme, according to a specified _Scope_.
 Here we will use the `users` collection within the `tenant_agent_00` scope from `travel-sample` bucket as an example.
 
-[source,golang,indent=0]
+[source,golang]
 ----
-include::example$startusing.go[tag=collection]
+include::devguide:example$go/cloud.go[tag=collection,indent=0]
 ----
 
-The code shows how you would use a named collection and scope. A named or default collection will provide the same functionality as bucket-level operations did in previous versions of Couchbase Server.
+The code shows how you would use a named collection and scope. 
+A named or default collection will provide the same functionality as bucket-level operations did in previous versions of Couchbase Server.
 
 The `DefaultCollection` must be used when connecting to a 6.6 cluster or earlier.
 
-[source,golang,indent=0]
+=== Document Addition and Retrieval
+
+Let's create a struct in our application that we can add to our `travel-sample` bucket that conforms to the structure of a document of type user.
+
+[source,golang]
+----
+include::devguide:example$go/cloud.go[tag=document,indent=0]
+----
+
+xref:howtos:kv-operations.adoc[Document Operations] operations, such as storing and retrieving documents, can be done using `Collection.Upsert()` and `Collection.Get()`.
+Simply pass the key (and value, if applicable) to the relevant methods.
+
+The following function will _upsert_ a document into the database:
+
+[source,golang]
+----
+include::devguide:example$go/cloud.go[tag=upsert,indent=0]
+----
+
+Now, let's retrieve that document using a key-value operation. 
+The following function runs a get for a document key and logs out the result in our console:
+
+[source,golang]
 ----
-include::example$startusing.go[tag=upsert-get]
+include::devguide:example$go/cloud.go[tag=get,indent=0]
 ----
 
-KV Operations are described in detail on the xref:howtos:kv-operations.adoc[KV Operations page].
-Now that you know the basics, you may wish to go straight to that page.
+=== SQL++ Lookup
```

Comment:
Letting this through - but we still await clear guidance on SQL++ use in place of N1QL for the docs.